extensions/commands/cci/cmd_list_v2_ready.py

```diff
@@ -39,7 +45,8 @@ def list_v2_ready(conan_api: conan.api.conan_api.ConanAPI, parser, *args):
         recipe_folder = os.path.join("recipes", recipe_name)
         config_file = os.path.join(recipe_folder, "config.yml")
         if not os.path.exists(config_file):
-            raise ConanException(f"The file {config_file} does not exist")
+            out.error(f"The file {config_file} does not exist")
```

Comment:
What recipes are missing the `config.yml` I would except all of them to have one 🤔

Reply:
it's not really needed because it was a way to say to jenkins which versions should be built and where they are located. On the beginning of CCI config.yml was more optional, because instead of `all` folder,  we used the version as folder name and Jenkins should be able to understand it.

Reply:
I am not sure about this. If it will fail if the file is not there in C3I, then shouldn't this be a hard error? I am fine with either, as long as you all have it clear.